content/_data/changelogs/weekly.yml

```diff
@@ -8823,6 +8823,40 @@
     message: |-
       Avoid warning on logs about Anonymous Class in hudson.FilePath.
 
+- version: '2.257'
+  date: 2020-09-15
+  changes:
+  - type: rfe
+    category: rfe
+    pull: 4920
+    issue: 60972
+    authors:
+    - romenrg
+    message: |-
+      Name the first breadcrumb "Dashboard" for clarity.
+  - type: bug
+    category: developer
+    pull: 4517
+    issue: 43889
+    authors:
+    - thomasgl-orange
+    message: |-
+      Developer: Remove test environments more consistently.
+  - type: rfe
+    category: developer
+    pull: 4517
+    issue: 43889
+    authors:
+    - thomasgl-orange
+    message: |-
+      Developer: New static utility method <code>Result#combine(Result,Result)</code> to get the worst of two (nullable) build results.
+
+  # pull: 4255 (PR title: IntelliJ/Java: Duplicate declaration -> ApiTokenProperty.checkPermission)
+  # pull: 4721 (PR title: Try to make the cron help clearer)
+  # pull: 4917 (PR title: Increase contrast of the disabled plugin uninstall button)
+  # pull: 4927 (PR title: Added localizaton for devsecops=DevSecOps)
+  # pull: 4930 (PR title: [JENKINS-63639] Update commons-io to 2.8.0)
```

Comment:
I thought we included dep updates as they sometimes causes regression?

Reply:
You're correct that we've included component updates (like remoting, groovy, and jetty) in prior changelogs.  [PR-4930](https://github.com/jenkinsci/jenkins/pull/4930) was labeled as `skip-changelog` and so the changelog generator placed it as a comment rather than including it in the changelog.

@MRamonLeon would you object if we placed it in the changelog?

@daniel-beck I don't see anything in the [styleguide](https://github.com/jenkins-infra/jenkins.io/blob/master/content/_data/changelogs/_STYLEGUIDE.adoc) that would prohibit mentioning the commons-io upgrade.  Any objections from you?

Reply:
No.

Reply:
+1